### PR TITLE
Condense Dragon's Blood Pudding effects (moderate and greater versions) and revise automation

### PR DESCRIPTION
**Pull request by ammalagonc**
Now grants quickened on all versions

### CODE REVIEW

File: packs/equipment-effects/effect-dragons-blood-pudding-major.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
     "name": "Effect: Dragon's Blood Pudding (Major)",
     "system": {
         "description": {
-            "value": "<p>Granted by @UUID[Compendium.pf2e.equipment-srd.Item.Dragon's Blood Pudding (Major)]</p>\n<p>You gain a +4 item bonus to the Intimidation check to Demoralize.</p>"
+            "value": "<p>Granted by @UUID[Compendium.pf2e.equipment-srd.Item.Dragon's Blood Pudding (Major)]</p>\n<p>You are quickened. You can use the additional action to Step, Stride, Strike, or Demoralize. You gain a +4 item bonus to the Intimidation check</p>"
```
Comment (pedrogrullada) [inline packs/equipment-effects/effect-dragons-blood-pudding-major.json:7]:
```suggestion
            "value": "<p>Granted by @UUID[Compendium.pf2e.equipment-srd.Item.Dragon's Blood Pudding (Major)]</p>\n<p>You are quickened. You can use the additional action to Step, Stride, Strike, or Demoralize. You gain a +4 item bonus to Intimidation checks to Demoralize.</p>"
```

File: packs/equipment-effects/effect-dragons-blood-pudding.json [hunk 2]
```diff
@@ -0,0 +1,96 @@
+{
+    "_id": "nbMEBxeqZIdOgysR",
+    "img": "systems/pf2e/icons/equipment/alchemical-items/alchemical-tapas/dragons-blood-pudding.webp",
+    "name": "Effect: Dragon's Blood Pudding",
+    "system": {
+        "description": {
+            "value": "<p>Granted by @UUID[Compendium.pf2e.equipment-srd.Item.Dragon's Blood Pudding (Moderate)], @UUID[Compendium.pf2e.equipment-srd.Item.Dragon's Blood Pudding (Greater)]</p>\n<p>You are quickened. You can use the additional action to Step, Stride, Strike, or Demoralize. You gain an item bonus to the Intimidation check</p>"
```
Comment (pedrogrullada) [inline packs/equipment-effects/effect-dragons-blood-pudding.json:7]:
```suggestion
            "value": "<p>Granted by @UUID[Compendium.pf2e.equipment-srd.Item.Dragon's Blood Pudding (Moderate)], @UUID[Compendium.pf2e.equipment-srd.Item.Dragon's Blood Pudding (Greater)]</p>\n<p>You are quickened. You can use the additional action to Step, Stride, Strike, or Demoralize. You gain an item bonus to Intimidation checks to Demoralize.</p>"
```

File: packs/equipment-effects/effect-dragons-blood-pudding-major.json
Comment (pedrogrullada) [inline packs/equipment-effects/effect-dragons-blood-pudding-major.json:1]: The level for this effect should be 19, if you wanna get that while you're at it

File: packs/equipment/dragons-blood-pudding-moderate.json [hunk 3]
```diff
@@ -11,7 +11,7 @@
...
+            "value": "<p><strong>Activate</strong> <span class=\"action-glyph\">1</span> Interact</p><hr /><p>While some unscrupulous chefs claim that this savory pudding is made with real dragon's blood, its crimson color and acrid smell actually come from bloody mandrake paste, ginger root, and distilled terrap sap. This potent combination singes the nostrils and throat, removing effects that make you sluggish. When you consume the pudding, it attempts a counteract check with a [[/r {1d20+9}#Counteract Check]]{+9} counteract modifier to remove the @UUID[Compendium.pf2e.conditionitems.Item.Slowed] condition from a single source, using the source of that condition to determine the counteract rank and DC. You also become @UUID[Compendium.pf2e.conditionitems.Item.Quickened] for 1 round. While taking the additional action from being quickened, you scream, as a result of both the flavor and the mandrake paste's properties. You can use the additional action to Step, Stride, Strike, or Demoralize. If you choose to Demoralize, you gain a +2 item bonus to the Intimidation check.</p>\n<p>@UUID[Compendium.pf2e.equipment-effects.Item.Effect: Dragon's Blood Pudding]</p>"
```
Comment (simonward) [inline packs/equipment/dragons-blood-pudding-moderate.json:14]:
```suggestion
            "value": "<p><strong>Activate</strong> <span class=\"action-glyph\">1</span> Interact</p><hr /><p>While some unscrupulous chefs claim that this savory pudding is made with real dragon's blood, its crimson color and acrid smell actually come from bloody mandrake paste, ginger root, and distilled terrap sap. This potent combination singes the nostrils and throat, removing effects that make you sluggish. When you consume the pudding, it attempts a counteract check with a [[/r {1d20+9}#Counteract Check]]{+9} counteract modifier to remove the @UUID[Compendium.pf2e.conditionitems.Item.Slowed] condition from a single source, using the source of that condition to determine the counteract rank and DC. You also become @UUID[Compendium.pf2e.conditionitems.Item.Quickened] for 1 round. While taking the additional action from being quickened, you scream, as a result of both the flavor and the mandrake paste's properties. You can use the additional action to Step, Stride, Strike, or @UUID[Compendium.pf2e.actionspf2e.Item.Demoralize]. If you choose to Demoralize, you gain a +2 item bonus to the Intimidation check.</p>\n<p>@UUID[Compendium.pf2e.equipment-effects.Item.Effect: Dragon's Blood Pudding]</p>"
```

File: packs/equipment/dragons-blood-pudding-greater.json [hunk 4]
```diff
@@ -11,7 +11,7 @@
...
+            "value": "<p><strong>Activate</strong> <span class=\"action-glyph\">1</span> Interact</p><hr /><p>While some unscrupulous chefs claim that this savory pudding is made with real dragon's blood, its crimson color and acrid smell actually come from bloody mandrake paste, ginger root, and distilled terrap sap. This potent combination singes the nostrils and throat, removing effects that make you sluggish. When you consume the pudding, it attempts a counteract check with a [[/r {1d20+20}#Counteract Check]]{+20} counteract modifier to remove the @UUID[Compendium.pf2e.conditionitems.Item.Slowed] condition from a single source, using the source of that condition to determine the counteract rank and DC. You also become @UUID[Compendium.pf2e.conditionitems.Item.Quickened] for 1 round. While taking the additional action from being quickened, you scream, as a result of both the flavor and the mandrake paste's properties. You can use the additional action to Step, Stride, Strike, or Demoralize. If you choose to Demoralize, you gain a +3 item bonus to the Intimidation check.</p>\n<p>@UUID[Compendium.pf2e.equipment-effects.Item.Effect: Dragon's Blood Pudding]</p>"
```
Comment (simonward) [inline packs/equipment/dragons-blood-pudding-greater.json:14]:
```suggestion
            "value": "<p><strong>Activate</strong> <span class=\"action-glyph\">1</span> Interact</p><hr /><p>While some unscrupulous chefs claim that this savory pudding is made with real dragon's blood, its crimson color and acrid smell actually come from bloody mandrake paste, ginger root, and distilled terrap sap. This potent combination singes the nostrils and throat, removing effects that make you sluggish. When you consume the pudding, it attempts a counteract check with a [[/r {1d20+20}#Counteract Check]]{+20} counteract modifier to remove the @UUID[Compendium.pf2e.conditionitems.Item.Slowed] condition from a single source, using the source of that condition to determine the counteract rank and DC. You also become @UUID[Compendium.pf2e.conditionitems.Item.Quickened] for 1 round. While taking the additional action from being quickened, you scream, as a result of both the flavor and the mandrake paste's properties. You can use the additional action to Step, Stride, Strike, or @UUID[Compendium.pf2e.actionspf2e.Item.Demoralize]. If you choose to Demoralize, you gain a +3 item bonus to the Intimidation check.</p>\n<p>@UUID[Compendium.pf2e.equipment-effects.Item.Effect: Dragon's Blood Pudding]</p>"
```

Comment (simonward) [reply packs/equipment/dragons-blood-pudding-greater.json:14]: Also add `@UUID[Compendium.pf2e.actionspf2e.Item.Demoralize]` link to major.

**Pull request merged**